ZelBack/src/services/fluxCommunication.js

```diff
@@ -52,7 +52,20 @@ async function isFluxAvailable(ip) {
   try {
     const fluxResponse = await serviceHelper.axiosGet(`http://${ip}:${config.server.apiport}/flux/version`, axiosConfig);
     if (fluxResponse.data.status === 'success') {
-      return true;
+      try {
+        const fluxBenchInfoResponse = await serviceHelper.axiosGet(`http://${ip}:${config.server.apiport}/benchmark/getinfo`, axiosConfig);
```

Comment:
Note: Need to be careful if the request of the other node won't time out as we now have 2 calls to the node which may exceed timeout limit of the node asking